homeassistant/components/dlna_dmr/config_flow.py

```diff
@@ -439,22 +439,14 @@ def _is_ignored_device(discovery_info: ssdp.SsdpServiceInfo) -> bool:
     manufacturer = (discovery_info.upnp.get(ssdp.ATTR_UPNP_MANUFACTURER) or "").lower()
     model = (discovery_info.upnp.get(ssdp.ATTR_UPNP_MODEL_NAME) or "").lower()
 
-    if manufacturer.startswith("xbmc") or model == "kodi":
-        # kodi
-        return True
-    if "philips" in manufacturer and "tv" in model:
-        # philips_js
-        # These TVs don't have a stable UDN, so also get discovered as a new
+    return (
+        (manufacturer.startswith("xbmc") or model == "kodi")  # kodi
+        # Philips TVs don't have a stable UDN, so also get discovered as a new
         # device every time they are turned on.
-        return True
-    if manufacturer.startswith("samsung") and "tv" in model:
-        # samsungtv
-        return True
-    if manufacturer.startswith("lg") and "tv" in model:
-        # webostv
-        return True
-
-    return False
+        or ("philips" in manufacturer and "tv" in model)  # philips_js
+        or (manufacturer.startswith("samsung") and "tv" in model)  # samsungtv
+        or (manufacturer.startswith("lg") and "tv" in model)  # webostv
+    )
```

Comment:
Here as well. Overall it looks like we should just disable this rule. It can't be applied consistently and often leads to worse code IMO.

Reply:
I don't have a problem with disabling the rule, because you are right, building big boolean constructions can lead to quite ugly code.